src/index.js

```diff
@@ -187,23 +191,38 @@ const renderLevel = (root, levelNumber) => {
   });
  
   const moves = {
+    KeyA: "L",
     ArrowLeft: "L",
+    KeyW: "U",
     ArrowUp: "U",
+    KeyD: "R",
     ArrowRight: "R",
+    KeyS: "D",
     ArrowDown: "D",
   };
+  const dirToDegree = {
+    KeyA: 270,
+    ArrowLeft: 270,
+    KeyW: 0,
+    ArrowUp: 0,
+    KeyD: 90,
+    ArrowRight: 90,
+    KeyS: 180,
+    ArrowDown: 180,
+  };
+  
   document.onkeydown = event => {
     if (event.code in moves) {
       event.preventDefault();
 
       if (soko.move(moves[event.code])) {
         renderBoard();
-
         if (soko.solved()) {
-          setTimeout(() => { renderLevelComplete(root); }, 100);
           console.log("solved!");
         }
       }
+      const playerEl = document.querySelector(".player, .player-on-goal");
+      playerEl.style.transform = `rotate(${dirToDegree[event.code]}deg)`;
```

Comment:
This code is repeated in 2 places and would be a good candidate to be put into its own function, `rotatePlayer`.